wordpress-mobile/WordPress-Android
# Feature/glide notification detail

Fixes #7958 

Not Ready for review - #8048 needs to be merged first as this branch is based on it.

Starts loading image in the notification detail with Glide.
![notif_detail](https://user-images.githubusercontent.com/2261188/43127982-199937fc-8f31-11e8-81d4-5dfd3e21d943.png)


To test:
1. Go to Notifications
2. Click on any "award" notification (such as "You've made X posts on XYZ")
3. Make sure the image is correctly loaded and the pop animation works

Note
I also wanted to start using glide (WpCustomImageGetter) for detail of comment notification, however there is a bug and the images are not loading at all. So I thought it would be better to fix the bug first and do the migration to Glide later. I submitted an issue #8098.
